yew-macro/tests/macro/html-tag-fail.stderr

```diff
@@ -136,6 +136,30 @@ error: this dynamic tag is missing an expression block defining its value
 46 |     html! { <@/> };
    |              ^
 
+error: boolean attributes don't support being used as an optional attribute (hint: a value of false results in the attribute not being set)
+  --> $DIR/html-tag-fail.rs:50:20
+   |
+50 |     html! { <input disabled?=Some(true) /> };
+   |                    ^^^^^^^^
+
+error: the `checked` attribute does not support being used as an optional attribute
+  --> $DIR/html-tag-fail.rs:58:20
+   |
+58 |     html! { <input checked?=Some(false) /> };
+   |                    ^^^^^^^
+
+error: the `class` attribute does not support being used as an optional attribute
+  --> $DIR/html-tag-fail.rs:59:20
+   |
+59 |     html! { <input class?=() /> };
```

Comment:
Why `class` can't be used as optional attribute?

Reply:
`class` already supports `Option<String>` as a value.